src/components/cc-orga-member-list/cc-orga-member-list.smart.md

```diff
@@ -0,0 +1,62 @@
+---
+kind: '🛠 Organisation/<cc-orga-member-list>'
+title: '💡 Smart'
+---
+
+# 💡 Smart `<cc-orga-member-list>`
+
+## ℹ️ Details
+
+<table>
+  <tr><td><strong>Component    </strong> <td><a href="https://www.clever-cloud.com/doc/clever-components/?path=/docs/🛠-organisation-cc-orga-member-list--default-story"><code>&lt;cc-orga-member-list></code></a>
+  <tr><td><strong>Selector     </strong> <td><code>cc-orga-member-list</code>
+  <tr><td><strong>Requires auth</strong> <td>Yes
+</table>
+
+## 👋️ Events fired
+
+| Name                              | Payload | Details                                                                                                       |
+|-----------------------------------|---------|---------------------------------------------------------------------------------------------------------------|
+| `cc-orga-member-list-smart:leave` |         | Fired when the current user leaves the organisation.<br/>Should be used to redirect the user to another page. |
```

Comment:
This is confusing, even I thought I made a mistake at the beginning but this is related to events fired by the smart component.

Maybe I should make it clearer with the name but I don't really know how. Maybe I could ditch the name of the component in the name and use something like "orga-member:leave"?
@hsablonniere If you have ideas for this?

Also, should we rename and use this section to list events that the smart component listens to as well?

Reply:
This is supposed to be a doc for users of the smart component so I wouldn't list the events listened by the smart component. I'm also thinking why not but if we want this, let's create an issue to discuss it.

We will probably remove the prefix of all of our events (we need to remove the `:` anyway) so I'm not sure yet.

I think this very event sent from smart could be prefix less and be `orga-member-leave-success`. I added a `-success` because it's fire only if the leave succeeded.

Reply:
Makes sense, thanks :+1: